 ERROR root [0x000035dc] [2018-05-11 00:07:57,501] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000035dc] [2018-05-11 00:07:57,530] - The following logs sent to logger failed:
  WARN root [0x000035dc] [2018-05-11 00:07:57,531] - ==> [2018-05-11 00:07:57 499] TCPPacketReceiver::processPendingEvents(10.1.9.9:30870): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000035dc] [2018-05-11 00:07:57,532] - ==> [2018-05-11 00:07:57 526] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000035dc] [2018-05-11 00:07:57,658] - TCPPacketReceiver::processPendingEvents(10.1.9.9:30873): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000035dc] [2018-05-11 00:07:57,663] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000035dc] [2018-05-11 00:07:57,664] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000035dc] [2018-05-11 00:07:57,670] - onBaseAppShutDown: state=0
  INFO root [0x000035dc] [2018-05-11 00:07:57,670] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000035dc] [2018-05-11 00:07:57,670] - onBaseAppShutDown: state=1
  INFO root [0x000035dc] [2018-05-11 00:07:57,670] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000035dc] [2018-05-11 00:07:57,671] - onBaseAppShutDown: state=2
 ERROR root [0x000035dc] [2018-05-11 00:07:57,702] - Equipment::onCellWriteToDBCompleted(1): not found dbmgr!
 ERROR root [0x000035dc] [2018-05-11 00:07:57,704] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,706] - Equipment::~Equipment(): 1
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,708] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,708] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,708] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,709] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,709] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,709] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,710] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,710] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000035dc] [2018-05-11 00:07:57,745] - Script::uninstall(): is successfully!
 DEBUG root [0x000035dc] [2018-05-11 00:07:57,746] - ThreadPool::destroy(): starting size 2.
